### PR TITLE
Add a default Issue template

### PR DESCRIPTION
**Pull request by mburns**
Simple template markdown for filing new Issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<!--- If you're requesting support, tell us what we can do for you -->
+<!--- If you're reporting a problem, tell us the situation so we can help -->
+
+## Requirements
+
+<!--- List any requirements or limits that need to be satisfied -->
+
+## Proposal
+
+<!--- Describe how you would like to see this solved -->
```
Comment (daviddias) [inline .github/ISSUE_TEMPLATE.md]: Is this the repo for support tickets? I thought it was https://github.com/protocol/infra

Comment (eefahy) [reply .github/ISSUE_TEMPLATE.md]: Actually I think the actual templates belong on ipfs/infrastructure & protocol/infra whereas this repo provides an expanded description on them

Comment (mburns) [reply .github/ISSUE_TEMPLATE.md]: https://github.com/ipfs/infrastructure/pull/457
https://github.com/protocol/infra/pull/400

Might make sense to clarify https://github.com/protocol/infra-team#creating-an-issue-page_facing_up or tweak this template to reflect it.

Comment (daviddias) [reply .github/ISSUE_TEMPLATE.md]: Small nitpick, can we rename protocol/infra to protocol/infrastructure so that the name is consistent across orgs?

Comment (eefahy) [reply .github/ISSUE_TEMPLATE.md]: ipfs/infrastructure is actually the outlier as all other repos are `infra` or `*-infra` How do you feel about renaming it to ipfs/infra?

Comment (daviddias) [reply .github/ISSUE_TEMPLATE.md]: Rename was done :)

Comment (eefahy) [reply .github/ISSUE_TEMPLATE.md]: Thank you :)

**Comment by eefahy** [comment]
Closing in favor of ipfs/infra#457 & protocol/infra#400

**Pull request closed without merging**